stdlib/3.4/pathlib.pyi

```diff
@@ -93,8 +95,12 @@ class Path(PurePath):
     if sys.version_info >= (3, 5):
         @classmethod
         def home(cls: Type[_P]) -> _P: ...
-        def __new__(cls: Type[_P], *args: Union[str, PurePath],
-                    **kwargs: Any) -> _P: ...
+        if sys.version_info < (3, 6):
+            def __new__(cls: Type[_P], *args: Union[str, PurePath],
+                        **kwargs: Any) -> _P: ...
```

Comment:
Interesting, apparently Path takes arbitrary kwargs and then ignores them. This doesn't appear to be documented. Do you know why this is? I might open a bug against CPython about this.